clonelists/Sony - PlayStation 2.json

```diff
@@ -2330,7 +2338,9 @@
 		],
 		"Psyvariar - Complete Edition (Special Capture Box)": [
```

Comment:
The convention when a disc contains two separate titles is to list the two individual titles, and set the compilation to zero priority:

```
		"SuperLite 2000 Vol. 8 - Psyvariar - Medium Unit": [
			["Psyvariar - Complete Edition", 0],
			["Psyvariar - Complete Edition (Special Capture Box)", 0],
			["Psyvariar - Complete Edition (Special Sound Box)", 0]
		],
		"SuperLite 2000 Vol. 10 - Psyvariar - Revision": [
			["Psyvariar - Complete Edition", 0],
			["Psyvariar - Complete Edition (Special Capture Box)", 0],
			["Psyvariar - Complete Edition (Special Sound Box)", 0]
		],
```

Reply:
Updated.